arch/x86/aerugo-x86-hal/src/hal.rs

```diff
@@ -32,14 +31,16 @@ impl AerugoHal for Hal {
     }
 
     fn get_system_time() -> Instant {
-        Instant::from_ticks(
+        let duration = Duration::nanos(
             TIME_START
                 .elapsed()
                 .expect("{}")
                 .as_nanos()
                 .try_into()
                 .unwrap(),
-        )
+        );
+
+        Instant::from_ticks(duration.ticks())
```

Comment:
why like that, readability, or is there some difference in code underneath?

Reply:
It was wrong before, it passed nanoseconds instead of ticks